hil/ext/switches/n3000.py

```diff
@@ -44,14 +41,13 @@ class DellN3000(Switch):
 
     @staticmethod
     def validate(kwargs):
-        """Note: Dell N3000 VLAN range only from 1 - 4093."""
-        Schema({
+        schema.Schema({
             'username': basestring,
             'hostname': basestring,
             'password': basestring,
-            'dummy_vlan': And(Use(int),
-                              lambda v: 0 < v <= 4093,
-                              Use(str)),
+            'dummy_vlan': schema.And(schema.Use(int),
+                                     lambda v: 0 < v and v <= 4093,
```

Comment:
Why get rid of the `0 < v <= 4093` shorthand?

Also, while I don't necessary have an objection to qualifying all of the variables from the `schema` module, I don't understand how it's related to this patch?

Reply:
I'll clean this PR up, I'm not sure how this change snuck in there.

Reply:
Actually this should be it for style-related things, the rest should be functional. I'm not sure why the diff is appearing a bit strangely in `_dell_base`